### PR TITLE
CalendarList does not implement `onMonthChange` #659

### PR DESCRIPTION
**Pull request by Jacklol**
https://github.com/wix/react-native-calendars/issues/659 solve this problem

### CODE REVIEW

**Comment by Jacklol** [comment]
can merge it or  tell  what wrong?

**Comment by webdevsyd** [comment]
I hope this PR can be merge immediately.

File: src/calendar-list/index.js [hunk 1]
```diff
@@ -283,6 +283,9 @@ class CalendarList extends Component {
           data={this.state.rows}
           //snapToAlignment='start'
           //snapToInterval={this.calendarHeight}
+          viewabilityConfig={{
+            itemVisiblePercentThreshold: 50
+          }}
```
Comment (Inbal-Tish) [inline src/calendar-list/index.js]: Currently the viewabilityConfig sets itemVisiblePercentThreshold to 20. I don't see how that or changing it to 50 will fix the issue with invoking onMonthChange

Comment (Jacklol) [reply src/calendar-list/index.js]: https://stackoverflow.com/questions/45868284/how-to-get-currently-visible-index-in-rn-flat-list

Comment (Jacklol) [reply src/calendar-list/index.js]: Its wark i checked

Comment (Jacklol) [reply src/calendar-list/index.js]: onVisibleMonthsChange={(months) => { console.log('now these months are visible', this.setMonths(months)); }}
          pagingEnabled={true}
 result =>{year: 2019, month: 2, day: 19, timestamp: 1550534400000, dateString: "2019-02-19"}

Comment (Jacklol) [reply src/calendar-list/index.js]: u can checked too

Comment (Jacklol) [reply src/calendar-list/index.js]: we need current month , but now  if(pagingEnabled={true})  
result => #659

Comment (Inbal-Tish) [reply src/calendar-list/index.js]: First, setting itemVisiblePercentThreshold to 50 is will give you the same behaviour as the FlatList's default. Second, we already have 'this.viewabilityConfig' defined in the c'tor and you're ignoring it. And last creating a new object in the render function is bad practice. Please run the library again and check if your fix is still relevant

Comment (Jacklol) [reply src/calendar-list/index.js]: U are right in last version all wark thanks!!!  i warked width version 1 . 68

Comment (Inbal-Tish) [reply src/calendar-list/index.js]: That's great. Can you please close the PR

**Pull request closed without merging**